sql/parser/sql.y

```diff
@@ -1575,8 +1575,7 @@ createdb_opt_item:
 // option will silently break. Best defense is to provide a regression test
 // exercising every such option, at least at the syntax level.
 createdb_opt_name:
-  IDENT {}
-| CONNECTION LIMIT {}
+  CONNECTION LIMIT {}
```

Comment:
Yes, we'll almost certainly support options for `CREATE DATABASE` at some point, but the options here are all postgres specific. Re-adding the grammar rules for parsing the cockroach specific options will not be difficult.